AsyncDisplayKit/ASMapNode.h

```diff
@@ -12,9 +12,9 @@
 @interface ASMapNode : ASImageNode
 
 /**
- The current region of ASMapNode. This can be set at any time and ASMapNode will animate the change. This property may be set from a background thread before the node is loaded, and will automatically be applied to define the region of the static snapshot (if .liveMap = NO) or the internal MKMapView (otherwise).
+ The current options of ASMapNode. This can be set at any time and ASMapNode will animate the change. This property may be set from a background thread before the node is loaded, and will automatically be applied to define the behavior of the static snapshot (if .liveMap = NO) or the internal MKMapView (otherwise).
  */
-@property (nonatomic, assign) MKCoordinateRegion region;
+@property (nonatomic, readwrite) MKMapSnapshotOptions *options;
```

Comment:
this should just say "strong" instead of readwrite (readwrite is the default anyway, and strong is actually the analogous to assign - it is the specifier of the memory behavior.  e.g. you could have an assign, readwrite)